Issue Solved #2 #8

17himanshu opened the pull request:
Issues Resolved : #2 #8 

added yarn and defined controllers, model, routes and db conneection

17himanshu committed:
added yarn functionality removed npm

17himanshu committed:
removed backend/.gitgnore

17himanshu commented:
@ravinder-chadha issue solved: #2  
Sir actually previous pr got closed maybe because of close with comment I didn't notice . I am now again raising this fresh pr

ravinder-chadha commented:
This is a follow up comment from the previous PR that you raised, I think you might have missed this [comment](https://github.com/iRootsOrg/CodeBase/pull/14#issuecomment-2196318915)
Please create the rest of the folder and file structures as well. So, that you are good to start with the next issue.

17himanshu committed:
Resolved merge conflicts

17himanshu committed:
updated yarn.lock and package.json

17himanshu commented:
Sir after we had our discussion I have added config folder which has db.js in which I connect to MongoDB, added controllers folder and added fileController.js in which I have defined uploadContrloller ;I have tested it using postman and is working as expected and also data is going into MongoDB , defined routes folder in which I defined file routes in fileRoutes and created models/fileModel.js 

I am attaching screenshots below: 
<img width="960" alt="Screenshot 2024-06-28 182918" src="https://github.com/iRootsOrg/CodeBase/assets/96365482/283bbe92-7993-48f8-871d-b7445c8e09c9">
![image](https://github.com/iRootsOrg/CodeBase/assets/96365482/7d96acc7-d619-4295-a471-80131158b63a)
